pkg/opensearch/response_parser_test.go

```diff
@@ -17,15 +17,20 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func createDataQueriesForTests(tsdbQueries map[string]string) []backend.DataQuery {
+type tsdbQuery struct {
+	refId string
+	body  string
+}
+
+func createDataQueriesForTests(tsdbQueries []tsdbQuery) []backend.DataQuery {
```

Comment:
Alternatively I could create a second helper that takes an array, but the map sort of implies that it should take multiple queries.